GBMClient remote doctest fails

<!--
We know asking good questions takes effort, and we appreciate your time.
Thank you.

Please be aware that everyone has to follow our code of conduct:
https://github.com/sunpy/sunpy/blob/master/CODE_OF_CONDUCT.rst

Also that these comments are hidden when you submit this github issue.

Please have a search on our GitHub repository to see if a similar issue has already been posted.
If a similar issue is closed, have a quick look to see if you are satisfied by the resolution.
If not please go ahead and open an issue!
-->
### Description
<!-- Provide a general description of the bug. -->
One docstring test in `fermi.py` file when ran with `pytest-remotedata` fails
### Expected behavior
<!-- What did you expect to happen. -->
Test should pass outputting 3 records.
### Actual behavior
<!--
What actually happened.
Was the output confusing or poorly described?
-->
Test fails with 0 records.
### Steps to Reproduce
<!--
Please include **code** that reproduces the issue whenever possible.
The best reproductions are self-contained scripts with minimal dependencies.
-->
Run this on terminal, being in sunpy directory;
```python
!pytest --remote-data=any sunpy/net/dataretriever/sources/fermi_gbm.py
```
it fails without returning any results
```python
============================= test session starts ==============================
platform linux -- Python 3.6.9, pytest-5.4.2, py-1.8.1, pluggy-0.13.1
rootdir: /content/sunpy, inifile: setup.cfg
plugins: remotedata-0.3.2, cov-2.9.0, openfiles-0.5.0, doctestplus-0.7.0, filter-subpackage-0.1.1, asdf-2.6.0, arraydiff-0.3, astropy-header-0.1.2, hypothesis-5.16.0, mock-3.1.0, typeguard-2.7.1
collected 1 item                                                               

sunpy/net/dataretriever/sources/fermi_gbm.py F                           [100%]

=================================== FAILURES ===================================
________ [doctest] sunpy.net.dataretriever.sources.fermi_gbm.GBMClient _________
027     Both of which can be accessed through the attrs `a.Resolution <sunpy.net.attrs.Resolution>`.
028     The default data type is CSPEC unless the user defines.
029 
030     Examples
031     --------
032     >>> from sunpy.net import Fido, attrs as a
033     >>> res = Fido.search(a.Time('2015-06-21 00:00', '2015-06-23 23:59'),
034     ...                   a.Instrument.gbm, a.Detector.n3,
035     ...                   a.Resolution.ctime) #doctest: +REMOTE_DATA
036     >>> print(res) #doctest: +REMOTE_DATA
Differences (unified diff with -expected +actual):
    @@ -1,10 +1,7 @@
     Results from 1 Provider:
     <BLANKLINE>
    -3 Results from the GBMClient:
    -     Start Time           End Time      Source Instrument Wavelength
    -------------------- ------------------- ------ ---------- ----------
    -2015-06-21 00:00:00 2015-06-23 23:59:00  FERMI        GBM        nan
    -2015-06-21 00:00:00 2015-06-23 23:59:00  FERMI        GBM        nan
    -2015-06-21 00:00:00 2015-06-23 23:59:00  FERMI        GBM        nan
    +0 Results from the GBMClient:
    +Start Time End Time Source Instrument Wavelength
    +---------- -------- ------ ---------- ----------
     <BLANKLINE>
     <BLANKLINE>

/content/sunpy/sunpy/net/dataretriever/sources/fermi_gbm.py:36: DocTestFailure
=========================== short test summary info ============================
FAILED sunpy/net/dataretriever/sources/fermi_gbm.py::sunpy.net.dataretriever.sources.fermi_gbm.GBMClient
============================== 1 failed in 9.18s ===============================
```


### System Details
<!--
We at least need to know the sunpy version you are using.
We provide a short function (``sunpy.util.system_info()``) that will provide most of the below information.
This step is optional but strongly recommended.
-->
- SunPy Version: '2.0rc2'
- Astropy Version: '4.1.rc1'
- Python Version: '3.7.6'
- OS information: 'Linux Ubuntu 18.04 LTS'

--- a/sunpy/net/dataretriever/sources/fermi_gbm.py
+++ b/sunpy/net/dataretriever/sources/fermi_gbm.py
@@ -119,7 +119,7 @@ class GBMClient(GenericClient):
                  attrs.Physobs: [('CSPEC', 'counts accumulated every 4.096 seconds in 128 energy channels for each detector.'),
                                  ('CTIME', 'counts accumulated every 0.256 seconds in 8 energy channels')],
                  attrs.Detector: [
-            (f"N{x}", f"GBM Detector short name for the detector NAI_{x:02}") for x in range(12)],
+            (f"n{x}", f"GBM Detector short name for the detector NAI_{x:02}") for x in range(12)],
             attrs.Resolution: [
             ("CSPEC", "CSPEC 128 channel spectra every 4.096 seconds."),
             ("CTIME", "CTIME provides 8 channel spectra every 0.256 seconds")]

